src/kbmod/work_unit.py

```diff
@@ -183,18 +227,22 @@ def from_fits(cls, filename):
 
             # Read the size and order information from the primary header.
             num_images = hdul[0].header["NUMIMG"]
-            if len(hdul) != 4 * num_images + 3:
+            n_constituents = hdul[0].header["NCON"]
+            if len(hdul) != (4 * num_images) + (2 * n_constituents) + 3:
                 raise ValueError(
                     f"WorkUnit wrong number of extensions. Expected "
-                    f"{4 * num_images + 3}. Found {len(hdul)}."
+                    f"{6 * num_images + 3}. Found {len(hdul)}."
                 )
 
-            # Read in all the image files.
-            per_image_wcs = []
+            # Misc. reprojection metadata
+            reprojected = hdul[0].header["REPRJCTD"]
+            heliocentric_distance = hdul[0].header["HELIO"]
```

Comment:
Do we want all of these new fields to be required? I'm thinking yes for most of them, but maybe not for the HELIO centric distance or the GEO_{I} distances. If the image has not been reprojected, then I'm not sure it makes sense to read them out of the header.

There are two different ways we could handle this:
1) Make them required if they have been reprojected:
```
if reprojected:
    heliocentric_distance = hdul[0].header["HELIO"]
    ...
else:
    heliocentric_distance = something else
```

or 2) Make each one optional.
```
If "HELIO" in hdul[0].header:
    heliocentric_distance = hdul[0].header["HELIO"]
else:
   ....
```

Reply:
Technically the HELIO and GEO_{i} aren't "required" by the WorkUnit, I just had it output `None` or whatever the fits header equivalent is to keep everything consistent. We could be more explicit about them being optional in `from_fits` tho

Reply:
Keeping everything consistent makes sense. Let's go with that.